app-config.yaml

```diff
@@ -135,3 +135,15 @@ dynamicPlugins:
         - bindTarget: 'remotePlugins.Scaffolder.scaffolderPlugin.externalRoutes'
           bindMap:
             registerComponent: 'catalogImportPlugin.routes.importPage'
+
+orchestrator:
```

Comment:
Changes on this file cannot be pushed to this branch. Could you please revert it?